service/__init__.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016, 2019 John J. Rofrano. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Package for the application models and service routes
+"""
+import logging
+from flask import Flask
+
+# NOTE: Do not change the order of this code
+# The Flask app must be created
+# BEFORE you import modules that depend on it !!!
+
+# Create the Flask aoo
+app = Flask(__name__)
+
+# Load Configurations
+app.config.from_object('config')
+
+# Import the routes After the Flask app is created
+from service import routes
```

Comment:
Does models need to be imported from service here as well?

Reply:
it could be another story, this one is a pure flask skeleton